docker/redis-example.conf

```diff
@@ -50,8 +50,8 @@
 # it will abort. It is possible to use multiple loadmodule directives.
 #
 loadmodule /redis-json/target/release/librejson.so
-loadmodule /redis-search/bin/linux-arm64v8-release/search/redisearch.so
-# loadmodule /redis-search/bin/linux-x64-release/search/redisearch.so
+# loadmodule /redis-search/bin/linux-arm64v8-release/search/redisearch.so
```

Comment:
Is this needed in this PR?

Reply:
Not really, but it needs to be like that on development branch. Other wise it only runs on mac ( is a line that arno needs to change so it works for him).